src/reference/01-General-Info/90-Changes/45-sbt-0.13-syntax-migration-guide.md

```diff
@@ -0,0 +1,105 @@
+---
+out: sbt-0.13-Syntax-Migration-Guide.html
+---
+
+## sbt 0.13 Syntax Migration Guide
+
+### Introduction
+
+Before sbt 0.13 (sbt 0.9 to 0.12) it was very common to see in builds the usage of two aspects of sbt:
+
+* the "fishbone" operator: `<<=`, `<+=`, `<++=`
+* the tuple enrichments (apply and map) for TaskKey's and SettingKey's (eg. `(foo, bar) map { (f, b) => ... }`)
+
+The release of sbt 0.13 (which was over 3 years ago!) introduced the `.value` macro which allowed for much
+easier to read and write code, effectively making both aspects redudant and they were removed from the official
+documentation.
+
+As they will be removed in upcoming release of sbt 1.0.0 we've deprecated them in sbt 0.13.13, and here we'll
+help guide you to how to migrate your code.
+
+### Migrating simple expressions
+
+With simple expressions such as:
+
```

Comment:
Also let's not do the vertical alignment thing since that's not done anywhere else in this doc.